src/background.js

```diff
@@ -11,6 +11,14 @@ function runShowMeTheCode(tabId, tab) {
         return urlMatches(/.*\/pull\/.*\/files/);
     };
 
+    function isCommitView() {
+        return urlMatches(/.*\/commit\/.*/);
```

Comment:
This is unlikely, but there is possibility that project name (i.e. Viridovix/commit/) will be caught by this regex